java/test/jmri/jmrix/AbstractMonPaneTestBase.java

```diff
@@ -46,28 +40,26 @@ public void tearDown() {
 
     @Test
     public void testConcreteCtor() {
-        Throwable thrown = catchThrowable( () -> GuiActionRunner.execute(() -> pane.initComponents()));
-        assertThat(thrown).isNull();
+        assertDoesNotThrow( () -> ThreadingUtil.runOnGUI( () -> pane.initComponents() ));
```

Comment:
I recommend that you use `runOnGUIWithJmriException()` instead of `runOnGUI()`.

`runOnGUIWithJmriException()` will throw JmriException and RuntimeException as is, while `runOnGUI()` will throw RuntimeException as an InvocationTargetException.

See:
https://docs.oracle.com/en/java/javase/17/docs/api/java.desktop/javax/swing/SwingUtilities.html#invokeAndWait(java.lang.Runnable)

Reply:
Thanks @danielb987 , this makes sense.